openc3/lib/openc3/models/plugin_model.rb

```diff
@@ -200,7 +200,7 @@ def self.install_phase2(plugin_hash, scope:, gem_file_path: nil, validate_only:
           end
           unless validate_only
             Logger.info "Installing python packages from requirements.txt with pypi_url=#{pypi_url}"
-            puts `/openc3/bin/pipinstall --no-warn-script-location -i #{pypi_url} -r #{File.join(gem_path, 'requirements.txt')}`
+            puts `/openc3/bin/pipinstall --no-warn-script-location -i #{pypi_url} --trusted-host #{URI.parse(pypi_url).host} -r #{File.join(gem_path, 'requirements.txt')}`
```

Comment:
I like having the option to set trusted-host but it shouldn't always be on.  That defeats SSL even when you have a good SSL connection.  Maybe an ENV to enable trusted host?

Reply:
I added an environment variable called `PIP_ENABLE_TRUSTED_HOST` and only apply `--trusted-host` if that environment variable is set.